src/patternfly/components/DualListSelector/dual-list-selector.scss

```diff
@@ -16,79 +16,77 @@ $pf-v5-c-dual-list-selector__item--MaxNesting: 10;
   --#{$dual-list-selector}--GridTemplateColumns--pane--MinMax--max: #{pf-size-prem(450px)};
 
   // Header
-  --#{$dual-list-selector}__header--MarginBottom: var(--#{$pf-global}--spacer--sm);
+  --#{$dual-list-selector}__header--MarginBottom: var(--pf-t--global--spacer--sm);
 
   // Title
-  --#{$dual-list-selector}__title-text--FontWeight: var(--#{$pf-global}--FontWeight--bold);
+  --#{$dual-list-selector}__title-text--FontWeight: var(--pf-t--global--font--weight--body--bold);
 
   // Tools
-  --#{$dual-list-selector}__tools--MarginBottom: var(--#{$pf-global}--spacer--md);
-  --#{$dual-list-selector}__tools-filter--tools-actions--MarginLeft: var(--#{$pf-global}--spacer--sm);
+  --#{$dual-list-selector}__tools--MarginBottom: var(--pf-t--global--spacer--md);
+  --#{$dual-list-selector}__tools-filter--tools-actions--MarginLeft: var(--pf-t--global--spacer--sm);
 
   // Menu
-  --#{$dual-list-selector}__menu--BorderWidth: var(--#{$pf-global}--BorderWidth--sm);
-  --#{$dual-list-selector}__menu--BorderColor: var(--#{$pf-global}--BorderColor--100);
+  --#{$dual-list-selector}__menu--BorderWidth: var(--pf-t--global--border--width--regular);
+  --#{$dual-list-selector}__menu--BorderColor: var(--pf-t--global--border--color--default);
+  --#{$dual-list-selector}__menu--BorderRadius: var(--pf-t--global--border--radius--small);
   --#{$dual-list-selector}__menu--MinHeight: #{pf-size-prem(200px)};
   --#{$dual-list-selector}__menu--MaxHeight: #{pf-size-prem(320px)};
 
   // List item
-  --#{$dual-list-selector}__list-item-row--FontSize: var(--#{$pf-global}--FontSize--sm);
-  --#{$dual-list-selector}__list-item-row--BackgroundColor: transparent;
-  --#{$dual-list-selector}__list-item-row--hover--BackgroundColor: var(--#{$pf-global}--BackgroundColor--200);
-  --#{$dual-list-selector}__list-item-row--focus-within--BackgroundColor: var(--#{$pf-global}--BackgroundColor--200);
-  --#{$dual-list-selector}__list-item-row--m-selected--BackgroundColor: var(--#{$pf-global}--BackgroundColor--200);
-  --#{$dual-list-selector}__list-item--m-ghost-row--BackgroundColor: var(--#{$pf-global}--BackgroundColor--100);
+  --#{$dual-list-selector}__list-item-row--FontSize: var(--pf-t--global--font--size--sm);
+  --#{$dual-list-selector}__list-item-row--BackgroundColor:  var(--pf-t--global--background--color--action--plain--default);
+  --#{$dual-list-selector}__list-item-row--hover--BackgroundColor: var(--pf-t--global--background--color--primary--hover);
+  --#{$dual-list-selector}__list-item-row--BorderRadius: var(--pf-t--global--border--radius--tiny);
+  --#{$dual-list-selector}__list-item-row--focus-within--BackgroundColor: var(--pf-t--global--background--color--primary--hover);
```

Comment:
We should be able to remove this var and just reuse the hover var forboth hover and focus styles